modules/standard/IO.chpl

```diff
@@ -4022,6 +4040,83 @@ proc channel.writebits(v:integral, nbits:integral):bool throws {
   return try this.write(new ioBits(v:uint(64), nbits:int(8)));
 }
 
+proc channel.writeBinary(arg:numeric, param endian:ioendian = ioendian.native) throws {
+  var kind: iokind;
```

Comment:
Is this variable used any more?  It looks like you're just using the enum values directly in the select statement.

Reply:
Leftover debris, removed.